panel/src/components/Forms/Input/CalendarInput.vue

```diff
@@ -119,21 +121,21 @@ export default {
 			return this.toDate().daysInMonth();
 		},
 		/**
-		 * Adjusted weekday number (Sunday is 7 not 0)
+		 * Weekday column number (1-7) the first day of the month
+		 * needs to be placed at
 		 * @returns {number}
 		 */
 		firstWeekday() {
-			const weekday = this.toDate().day();
-			return weekday > 0 ? weekday : 7;
+			const day = days[this.toDate().day()];
+			return this.weekdays.indexOf(day) + 1;
 		},
 		/**
-		 * Translated weekday names
+		 * Ordered weekday names abbreviations
 		 * @returns {array}
 		 */
 		weekdays() {
-			return ["mon", "tue", "wed", "thu", "fri", "sat", "sun"].map((day) =>
-				this.$t("days." + day)
-			);
+			const first = this.$panel.config.weekday - 1;
```

Comment:
`Discuss`: should this be a prop to easily have Lab examples? Or is that overdoing it?